openapi/components/schemas/KycDocument/AddressMatches.yaml

```
@@ -70,3 +70,7 @@ properties:
     nullable: true
     allOf:
       - $ref: ./KycDocumentSubtypes.yaml
+  isTampered:
+    description: Specifies if an address proof document is suspicious of being tampered.
```

nitpick: 'suspicious' doesn't really work here

We should use this form if we are certain that the document has been tampered with 
```suggestion
    description: Specifies if an address proof document has been tampered with.
```
We should use this form if it's not certain that the document has been tampered with 
```suggestion
    description: Specifies if an address proof document may have been tampered with.
```